contrib/kube-prometheus/jsonnetfile.lock.json

```diff
@@ -28,7 +28,7 @@
                     "subdir": ""
                 }
             },
-            "version": "0669b548b8bc981f2676e7ec70c8f4a05fa39aa7"
+            "version": "3991660410dab201bb1f60b84d26e027c6c877e4"
```

Comment:
Is this necessary?

Reply:
Not strictly, but we need to update anyways.